wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java

```diff
@@ -190,21 +190,25 @@ else if (currentParam.matches(WSSQLPORTPATTERN))
                 System.out.println("If missing dependancies arise, test will halt!");
                 System.out.println("    otel.traces.exporter sys property: "+System.getProperty("otel.traces.exporter"));
                 System.out.println("    OTEL_TRACES_EXPORTER Env var: " + System.getenv("OTEL_TRACES_EXPORTER"));
-                System.out.println("        OTEL_TRACES_SAMPLER Env var: " + System.getenv("OTEL_TRACES_SAMPLER"));
-                System.out.println("        otel.traces.sampler sys property: "+System.getProperty("otel.traces.sampler"));
+                System.out.println("    OTEL_TRACES_SAMPLER Env var: " + System.getenv("OTEL_TRACES_SAMPLER"));
+                System.out.println("    otel.traces.sampler sys property: "+System.getProperty("otel.traces.sampler"));
                 System.out.println("    otel.logs.exporter: "+ System.getProperty("otel.logs.exporter"));
                 System.out.println("    OTEL_LOGS_EXPORTER Env var: " + System.getenv("OTEL_LOGS_EXPORTER"));
                 System.out.println("    otel.metrics.exporter: "+ System.getProperty("otel.metrics.exporter"));
                 System.out.println("    OTEL_METRICS_EXPORTER Env var: " + System.getenv("OTEL_METRICS_EXPORTER"));
 
-                globalOTel = AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();
-            }
-            else
-            {
-                globalOTel = GlobalOpenTelemetry.get();
+                if (Utils.isOtelJavaagentUsed())
```

Comment:
Should we add an env variable to override to turn this behavior off? IE: If we get a false positive from a non-otel java agent and the user needs to enable the AutoSDK.